src/TestWorkers-Client/TestResult.extension.st

```diff
@@ -23,12 +23,3 @@ TestResult >> basicSkipped [
 	
 	^ skipped
 ]
-
-{ #category : #'*TestWorkers-Client' }
-TestResult >> mergeWith: aTestResult [ 
```

Comment:
Is the sender of this message changed (TWClient)?

Thanks

Reply:
No. Actually it shouldn't be removed... I'll change this back

Reply:
Actually it is now in the main image (see https://github.com/pharo-project/pharo/commit/88889c6dcdb0fe12810ee8a63d2570840ec674d3)